misc/sync-typeshed.py

```diff
@@ -185,7 +185,21 @@ def main() -> None:
         "9f3bbbeb1",  # ParamSpec for functools.wraps
     ]
     for commit in commits_to_cherry_pick:
-        subprocess.run(["git", "cherry-pick", commit], check=True)
+        try:
+            subprocess.run(["git", "cherry-pick", commit], check=True)
+        except subprocess.CalledProcessError:
+            if not sys.__stdin__.isatty():
+                # We're in an automated context
+                raise
+
+            # Allow the option to merge manually
+            print(
+                f"Commit {commit} failed to cherry pick."
+                " In a separate shell, please manually merge and continue cherry pick."
+            )
+            rsp = input("Did you finish the cherry pick? [y/N]: ")
+            if not rsp.lower().startswith("y"):
```

Comment:
Could possibly be even stricter here, but this is very nitpicky!

```suggestion
            if rsp.lower() not in {"y", "yes"}:
```